Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -801,6 +805,49 @@ def image_from_vtk_image(vtk_image: "vtk.vtkImageData") -> "itkt.ImageBase":
     return l_image
 
 
+def dict_from_image(image: "itkt.Image") -> Dict:
+    """Serialize a Python itk.Image object to a pickable Python dictionary."""
+    import itk
+
+    pixel_arr = itk.array_view_from_image(image)
+    dimension = image.GetImageDimension()
+    componentType, pixelType = image_to_type(image)
+    if 'int64' in componentType:
+        # JavaScript does not yet support 64-bit integers well
+        if componentType == 'uint64_t':
+            pixel_arr = pixel_arr.astype(np.uint32)
+            componentType = 'uint32_t'
+        else:
+            pixel_arr = pixel_arr.astype(np.int32)
+            componentType = 'int32_t'
+    imageType = dict(
+        dimension=dimension,
+        componentType=componentType,
+        pixelType=pixelType,
+        components=image.GetNumberOfComponentsPerPixel()
+    )
+    return dict(
+        imageType=imageType,
+        origin=tuple(image.GetOrigin()),
+        spacing=tuple(image.GetSpacing()),
+        size=tuple(image.GetBufferedRegion().GetSize()),
+        direction=np.asarray(image.GetDirection()),
+        data=pixel_arr
+    )
+
+
+def image_from_dict(image_dict: Dict) -> "itkt.Image":
+    """Deserialize an dictionary representing an itk.Image object."""
+    import itk
+
+    ImageType = image_type_from_wasm_type(image_dict['imageType'])
```

Comment:
The return here is `(ImageType, dtype)` until `image_type_from_wasm_type` is updated to only return `ImageType`

Reply:
Ah yes, I should have remembered that. Thanks!